README.rst

```diff
@@ -41,6 +41,26 @@ To run all of the default test environments::
 Running the Server
 ~~~~~~~~~~~~~~~~~~
 
+--------------------
+With docker-compose
+--------------------
+
+Run::
+
+    # First time setup; these files are mounted into the container
+    cp conf/imsd-docker-compose-sample.conf conf/imsd.conf
+    cp conf/directory-sample.yaml conf/directory.yaml
```

Comment:
If we are starting MySQL up we probably should also use that for the directory, but it would be a pain to set up from scratch.

We need a pair of dump files to add in this process: one for a starting directory and another for a IMS DB with something to fiddle with.

I guess that could be a follow-on PR; no need to block here.